server/templates/place.html

```diff
@@ -58,11 +58,7 @@ <h4 id="place-dcid" class="mt-3 mt-m-0"><span><a href="/browser/{{place_dcid}}">
                 endtrans %} ›</a></span></h4>
         </div>
       </div>
-      {% if place_summary %}
-      <div id="summary" class="row-col">
-        <p>{{ place_summary }}</p>
```

Comment:
For a future PR: I wonder if leaving the text inline on page load would help with the soft 404 indexing errors. is it possible to run some sort of experiment to see if that helps?